components/Mixer.jsx

```diff
@@ -11,6 +11,39 @@ export default function Mixer() {
         width={240}
         height={520}
       />
+      <div className={styles.insideMixer}>
+        <div className={`${styles.thirdFruit} ${styles.fruitInMixer}`}>
+          {props.fruitsInMixer[2] != null && (
+            <Image
+              src={`/images/fruits/${props.fruitsInMixer[2]}.svg`}
+              alt=""
```

Comment:
Can you add fruit name in alt?

Reply:
Ok

Reply:
But, why didn't  you add alternative names in FruitIngredients components?

Reply:
Next to the fruit image we have fruit name so it's unnecessary